### DISCUSSION TITLE
does 'git lfs install' need to be run on every repo or just once on the computer (Windows 11)?

### DISCUSSION THREAD

**Discussion by mostlyhuman** [git-lfs/git-lfs#5096] [Q&A] [answered]
Thanks in advance!

**Comment by chrisd8088** [comment, marked as answer]
It should only need to be run once on the computer, and if you're using the Git for Windows [package](https://gitforwindows.org/), that step should be done for you by the installer, I believe.

**Reply by mostlyhuman** [reply]
Perfect! Thank you so much, Chris.
